### PR TITLE
Adding OperationalInsights cmdlets

### PR DESCRIPTION
**Pull request by pilor**
This is the initial check-in for the Operational Insights cmdlets based on the 0.9.0-preview SDK version. It contains all of Operational Insight's current management operations.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pilor__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cheggert). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1 [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 05/28/2015
+#
+
+@{
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
```
Comment (ogail) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1]: Fix module version to match current ones

File: src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1 [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = @()
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module
+ScriptsToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+FormatsToProcess = @(
+    '..\..\..\Package\Debug\ResourceManager\AzureResourceManager\OperationalInsights\Microsoft.Azure.Commands.OperationalInsights.format.ps1xml'
```
Comment (ogail) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1]: Check how to refer this in other psd1 files

Comment (pilor) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1]: @ogail Can you elaborate on this?  This psd1 is only used as input to the tool that generates the dll-help.xml.  It is set up the same way as other RPs dll-Help.psd1 files which are not referenced from anywhere besides help generation.

Comment (ogail) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Microsoft.Azure.Commands.OperationalInsights.dll-Help.psd1]: My bad, didn't notice its the help psd1

**Pull request merged**